pepper-apis/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -601,8 +603,14 @@ void enrichWithDSMEventDates(Handle handle,
         dataset.forEach(participant -> {
             DateValue diagnosisDate = medicalRecordService
                     .getDateOfDiagnosis(handle, participant.getUser().getId(), studyId).orElse(null);
-            DateValue birthDate = medicalRecordService
-                    .getDateOfBirth(handle, participant.getUser().getId(), studyId).orElse(null);
+
+            // First grab from the profile, if it's not there then look in activity data using medical-record service
```

Comment:
Some studies, like Osteo, will have the birth date copied into the profile, so I think we should grab from there as a more "canonical" source. And then fallback to `MedicalRecordService` for studies that doesn't have copying (like Angio/Brain).